wiki/redirect.yaml

```diff
@@ -1044,7 +1044,9 @@
 "scorev1":                     "Score/ScoreV1"
 "scorev2":                     "Score#scorev2"
 
-"skr":                         "Shortcut_Key_Reference"
```

Comment:
never remove redirects cuz u dont know if they are still used somewhere

Reply:
I was pretty sure I made this one up, but alright